lib/livebook_web/live/session_live.ex

```diff
@@ -741,7 +741,7 @@ defmodule LivebookWeb.SessionLive do
         case Runtime.duplicate(runtime) do
           {:ok, new_runtime} ->
             Session.connect_runtime(socket.assigns.session.pid, new_runtime)
-            socket
+            clear_flash(socket, :error)
```

Comment:
@josevalim in case we have show some other error message we would clear it too, but since the user already takes different action it seems alright to me. wdyt?

Reply:
Sounds good to me too!